src/utils.ts

```diff
@@ -0,0 +1,3 @@
+import config from 'config';
+
+export const prefix = `${config.get('SchemaRegistry.Protocol')}://${config.get('SchemaRegistry.Hostname')}:${config.get('SchemaRegistry.Port')}/apis/registry`;
```

Comment:
"utils.ts", can it be named differently? (Name "utils" is a red flag for me :slightly_smiling_face: ).

Reply:
Calling it `registry` as it is related to the schema registry